App.js

```diff
@@ -90,6 +110,8 @@ const App = () => {
         setVolunteer,
         installationId,
         setInstallationId,
+        submitOrder,
+        updateCart,
```

Comment:
You are passing this into the volunteer provider. I think you want it in the user provider